src/stream/pipeline/runner.rs

```diff
@@ -80,58 +78,60 @@ impl PipelineRunner {
-                                break;
-                            }
-                            lost_timestamps = 0;
-                        }
-
-                        Some(position)
-                    }
-                    None => Some(position),
-                }
-            }
+            // if pipeline.current_state() == gst::State::Playing {
+            //     if let Some(position) = pipeline.query_position::<gst::ClockTime>() {
+            //         previous_position = match previous_position {
+            //             Some(current_previous_position) => {
+            //                 if current_previous_position.nseconds() != 0
+            //                     && current_previous_position.nseconds() == position.nseconds()
+            //                 {
+            //                     lost_timestamps += 1;
+            //                     warn!("Position did not change {lost_timestamps}");
+            //                 } else {
+            //                     // We are back in track, erase lost timestamps
+            //                     lost_timestamps = 0;
+            //                 }
+
+            //                 if lost_timestamps > max_lost_timestamps {
+            //                     warn!("Pipeline lost too many timestamps (max. was {max_lost_timestamps}).");
+            //                     let _ = pipeline.set_state(gst::State::Null);
+            //                     // if let Err(error) = wait_for_element_state(
+            //                     //     pipeline.upcast_ref::<gst::Element>(),
+            //                     //     gst::State::Null,
+            //                     //     100,
+            //                     //     2,
+            //                     // ) {
+            //                     //     error!("Failed setting Pipeline {pipeline_id} to Null state. Reason: {error:?}");
+            //                     //     break;
+            //                     // }
+            //                     let _ = pipeline.set_state(gst::State::Playing);
+            //                     // if let Err(error) = wait_for_element_state(
+            //                     //     pipeline.upcast_ref::<gst::Element>(),
+            //                     //     gst::State::Playing,
+            //                     //     100,
+            //                     //     2,
+            //                     // ) {
+            //                     //     error!("Failed setting Pipeline {pipeline_id} to Playing state. Reason: {error:?}");
+            //                     //     break;
+            //                     // }
+            //                     lost_timestamps = 0;
+            //                 }
+
+            //                 Some(position)
+            //             }
+            //             None => Some(position),
+            //         }
+            //     }
+            // }
```

Comment:
This is one part that I need to revisit. I commented this because it acts against the thumbnail when blocking frames.